llvm/include/llvm/CodeGen/TargetLowering.h

```diff
@@ -1454,6 +1454,28 @@ class TargetLoweringBase {
            getLoadExtAction(ExtType, ValVT, MemVT) == Custom;
   }
 
+  /// Same as getLoadExtAction, but for atomic loads.
+  LegalizeAction getAtomicLoadExtAction(unsigned ExtType, EVT ValVT,
+                                        EVT MemVT) const {
+    if (ValVT.isExtended() || MemVT.isExtended()) return Expand;
```

Comment:
Line break

Reply:
This is copied from getLoadExtAction() a few lines up in the file, so maybe do the same..? (Or reformat both, I guess)